src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -460,7 +460,7 @@
     <value>A call is made to the default (parameterless) constructor of an exception type that is or derives from ArgumentException, or an incorrect string argument is passed to a parameterized constructor of an exception type that is or derives from ArgumentException.</value>
   </data>
   <data name="InstantiateArgumentExceptionsCorrectlyMessageNoArguments" xml:space="preserve">
-    <value>Call the {0} constructor that contains a message and/or paramName parameter.</value>
+    <value>Please use overload of the {0} constructor that contains a message and/or paramName parameter.</value>
```

Comment:
We are generally more assertive in the diagnostic message.
```suggestion
    <value>Use overload of the '{0}' constructor that contains a message and/or paramName parameter.</value>
```

Reply:
I recommend just reverting to the original because "Use" vs "Call" isn't strong enough to warrant a localization reset.

Reply:
Reverted but localization state still updated to `state="needs-review-translation"`